### ISSUE TITLE
Add a workbone-like UI

### ISSUE THREAD

**Issue by rkeene** [zankich/groku#7]
Add a text-based UI similar to workbone.  See attached script as an example.
[groku-ui.txt](https://github.com/zankich/groku/files/462130/groku-ui.txt)

**Comment by zankich** [comment]
Yeah this would be super nice to have, it would be a good subcommand
